src/index.js

```diff
@@ -0,0 +1,217 @@
+    const commentsData = await response.json();
+    const commentTitleEl = document.querySelector('.comments-title');
+    commentTitleEl.textContent = `Comments (${commentsData.length})`;
+    commentsData.forEach((comment) => {
+      const commentEl = document.createElement('p');
+      commentEl.classList.add('popup-comment');
+      commentEl.textContent = `${comment.creation_date} ${comment.username}: ${comment.comment}`;
+      popupCommentsEl.appendChild(commentEl);
+    });
+    const commentEl = document.createElement('p');
+    commentEl.classList.add('popup-comment');
+    commentEl.textContent = `${commentsData[1].creation_date} ${commentData.username}: ${commentData.comment}`;
+    popupCommentsEl.appendChild(commentEl);
+  });
+};
+
+const closePopup = (id) => {
+  const closeBtns = document.querySelectorAll('.popup-close-btn');
+  const commentsPopupWrapperEl = document.querySelector('.comments-popup-wrap');
+  const commentsPopupEl = document.querySelector(`#popup${id}`);
+  closeBtns.forEach((closeBtn) => {
+    closeBtn.addEventListener('click', () => {
+      commentsPopupEl.remove();
+      commentsPopupWrapperEl.style.display = 'none';
+    });
+  });
+};
+
+const renderCommentsPopup = (data) => {
+  const commentsPopupWrapperEl = document.querySelector('.comments-popup-wrap');
+  const commentsPopupEl = document.createElement('div');
+  commentsPopupEl.classList.add('comments-popup');
+  commentsPopupEl.id = `popup${data.id}`;
+  commentsPopupEl.innerHTML = `
+    <div class="popup-header">
+      <img class='popup-image' src='${data.image.original}'></img>
+      <span class='popup-close-btn'>&times;</span>
+    </div>
+    <div class='popup-content'>
+      <h2 class='title'>${data.name}</h2>
+      <div class='details'>
+        <div class='pop-show-header'>
+          <div class='pop-show-genre'>
+            <p class='show-genre'>Genres: ${data.genres}</p>
+            <p class='show-rating'>Rating: ${data.rating.average}</p>
+          </div>
+          <div class='pop-runtime-header'>
+            <p class='show-runtime'>Runtime: ${data.runtime}mins</p>
+            <p class='show-status'>Status: ${data.status}</p>
+          </div>
+        </div>
+        <p class='show-language'>Language: ${data.language}</p>
+        <p class='show-synopsis'>${data.summary}</p>
+      </div>
+    </div>
+    <h3 class='comments-title'>Comments (0)</h3>
+    <div class='popup-comments'></div>
+    <form class='popup-comment-form' id='popup-comment-form'>
+      <input type='name' name='name' placeholder='Your name' required>
+      <textarea name='comment' placeholder='Your comment' required></textarea>
+      <button type='submit'>Submit</button>
+    </form>
+  `;
+  commentsPopupWrapperEl.appendChild(commentsPopupEl);
+  closePopup(data.id);
+  getAndUpdateComment(data);
+  commentsCounter('.popup-comment');
+};
```

Comment:
- [ ] Nice work on instantly showing the comment rendered on the screen but there is a small issue with the comment count update. The comment count does not get updated instantly just as the comment text shows in an instant after making comment, unless after refreshing the page before the comment count number increase. Kindly work on the counter to get increased instantly when a new comment is added to the comments 👍🏽

Reply:
Thanks. We'll fix that

Reply:
fixed